integration/tests/access/cohort3/grpc_streaming_blocks_test.go

```diff
@@ -187,8 +182,12 @@ func (s *GrpcBlocksStreamSuite) TestHappyPath() {
 					foundANTxCount++
 				case block := <-observerBlocks:
 					s.T().Logf("ON block received: height: %d", block.Header.Height)
-					r.Add(s.T(), block.Header.Height, "observer", block)
-					foundONTxCount++
+
+					// check only that responses for ON which was tracked by AN and compare them
```

Comment:
```suggestion
					// the response tracker expects to receive data for the same heights from each node.
					// when subscribing to the latest block, the specific start height depends on the node's
					// local sealed height, so it may vary.
					// check only the responses for ON that are also tracked by AN and compare them
```